.github/workflows/issue-tests.yml

```diff
@@ -50,7 +58,7 @@ jobs:
         go test -v ./... -tags 'issues_integration'
 
     - name: Test Latest Stable YB Version
-      if: ${{ matrix.version == '2024.2.0.0-b145' }}
+      if: ${{ matrix.version == 'LATEST_STABLE' }}
```

Comment:
Also, we're doing all this just because we don't want this to be run in every matrix version, right? 
Can we just add this to another workflow? maybe misc-tests?

Reply:
As per my understanding, the matrix in this workflow is a list of voyager version, frequently updated with YB version, including latest ones which we want to test all the issues against.

So why create a new workflow? that will increase the work to maintain it and chances of missing to update.

Reply:
I think what Aneesh is saying it to shift this `Test Latest stable` one to another already existing workflow that doesn't have matrix.versions or runs as a single workflow so that we don't have to do this check.
We don't use this matrix version information for this particular test.

Reply:
Ok got your point.

Reply:
This new code is providing a placeholder to update the latest stable release and kind of enforcing the developers to update it when required so the issues-against thing also run for latest stable.

I see some value in that.
What do you guys think?